specification/refarch.adoc

```diff
@@ -3,142 +3,184 @@
-authenticated (per cryptographic signature mechanisms) by the TSM-driver 
-during the loading process, so that the TSM used is reflected in the 
-attestation rooted in a HW RoT. The authentication process provides 
-additional control to restrict TSM binaries that can be loaded on the 
-platform based on policies such as version, vendor etc. In addition to the 
-measurements, a security version number (SVN) of the TSM should be recorded 
-by the TSM-driver into the firmware measurement registers accessible only 
-to the TSM-driver and higher privilege components. The measurements and 
-versions of the HW RoT, the TSM-driver and the TSM will subsequently be 
-provided as evidence of a specific TSM being loaded on a specific platform. 
-
-During initialization, the TSM-driver will initialize a TSM-data region 
-within the TSM-memory region. The TSM-data region may hold per-hart TSM 
-state, memory assignment tracking structures and additional global data for 
-TSM management. The TSM-data region is TEE-capable memory that is apriori 
-access-control-restricted by the TSM-driver to allow only the TSM to access 
-this memory. The per-hart TSM state is used to start TSM execution from a 
-known-good state for security routines invoked by the OS/VMM. The per-hart 
-TSM state should be stored in pages that form a TSM Hart Control Structure 
+During platform initialization, HW elements form the RTM that measure the
+TSM-driver. The TSM-driver acts as the RTM for the TSM loaded on the
+platform. The TSM-driver initializes the TSM-memory-region for the TSM -
+this TSM-memory-region must be in TEE-capable memory. The TSM binary may be
+provided by the OS/VMM which may independently authenticate the binary
+before loading the binary into the TSM-memory-region via the TSM-driver.
+Alternatively, the firmware may pre-load the TSM binary via the TSM-driver.
+In both cases, the TSM binary loaded must be measured and may be
+authenticated (per cryptographic signature mechanisms) by the TSM-driver
+during the loading process, so that the TSM used is reflected in the
+attestation rooted in a HW RoT. The authentication process provides
+additional control to restrict TSM binaries that can be loaded on the
+platform based on policies such as version, vendor etc. In addition to the
+measurements, a security version number (SVN) of the TSM should be recorded
+by the TSM-driver into the firmware measurement registers accessible only
+to the TSM-driver and higher privilege components. The measurements and
+versions of the HW RoT, the TSM-driver and the TSM will subsequently be
+provided as evidence of a specific TSM being loaded on a specific platform.
+
+During initialization, the TSM-driver will initialize a TSM-data region
+within the TSM-memory region. The TSM-data region may hold per-hart TSM
+state, memory assignment tracking structures and additional global data for
+TSM management. The TSM-data region is TEE-capable memory that is apriori
```

Comment:
TEE-capable or confidential? May want to use a consitent terminology.

Reply:
fixed to use confidential